### PR TITLE
Feature/gradle support

### PR DESCRIPTION
**Pull request by tomaszmichalak**

### CODE REVIEW

**Comment by tomaszmichalak** [comment]
An example project is a pattern showing how to build a custom project distribution using the Knot.x Stack and project specific modules. We have projects that use both gradle and maven - it is much better to provide a gradle build script instead of dealing with one project on the other.

With this change projects that use gradle can easily adapt the example project into their codebase.

File: gradlew [hunk 1]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (tomaszmichalak) [inline gradlew]: Documentation should be updated.

**Comment by Skejven** [comment]
@marcinczeczko 
I understand this project as the example of how Knot.x-based platform codebase could look like.
And this example contains both, maven or gradle build scripts to show that both can be used to manage the project.
What is missing, is maybe an explanation of that idea in the main `README` file.

**Comment by tomaszmichalak** [comment]
@marcinczeczko @Skejven I am working on documentation.

**Pull request merged**